### PR TITLE
Create Bitsadmin Activity.yaml

### PR DESCRIPTION
**Pull request by mjmelone**
Change(s):
   - New Query

   Reason for Change(s):
   - No preexitsting query for bitsadmin activity

   Version Updated:
   - N/A

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @aprakash13, please review above hunting query PR and provide your feedback accordingly.

**Comment by v-spadarthi** [comment]
Hi @aprakash13, please review above hunting query PR and provide your feedback accordingly.

File: Hunting Queries/Microsoft 365 Defender/Execution/Bitsadmin Activity.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- CommandAndControl
+- Exfiltration
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceProcessEvents
+query: |
+  DeviceProcessEvents
+  | where 
+      (FileName =~ "bitsadmin.exe" or ProcessVersionInfoOriginalFileName =~ 'bitsadmin.exe')
```
Comment (shainw) [inline Hunting Queries/Microsoft 365 Defender/Execution/Bitsadmin Activity.yaml:20]: It appears that ProcessVersionInfoOriginalFileName is not available in the schema that is brought through to Sentinel.  Can you double check your query in Sentinel instance?

**Review by shainw — CHANGES_REQUESTED** [review]

**Comment by v-atulyadav** [comment]
Hi @mjmelone , can you please address the comment provided by @shainw. Thanks

**Comment by mjmelone** [comment]
@shainw code updated to work in both hunting interfaces.

**Comment by shainw** [comment]
> @shainw code updated to work in both hunting interfaces.

@mjmelone - Thanks for the workaround.  We will work on finding out why the column is not available. :)

**Review by shainw — APPROVED** [review]

**Pull request merged**